.bazelci/android-studio.yml

```diff
@@ -10,3 +24,54 @@ platforms:
       - --test_output=errors
     test_targets:
       - //:aswb_tests
+  Android-Studio-internal-canary:
```

Comment:
This is typically targeting an actual Canary version of Android Studio and hence is different to the -canary version aliases for IntelliJ CE/UE. We could consider to use different task names to make this clear (e.g. under-dev for IntelliJ CE/UE).

What I'm currently not sure about is whether this canary version correctly builds for the Bazel plugin at all times as it's a preliminary version after all. We might want to also use a soft fail strategy here just in case.

Reply:
I updated it.